cmd/machine-controller-manager-cli/METAL.md

```diff
@@ -0,0 +1,22 @@
+# Metal usage
```

Comment:
This file should be moved to `/docs/development`. And could we also make it generic enough to make instructions generic across different providers?  You could leave a line with the example, for now, to run metal maybe?